Description: "Bug-Debian:" field should be Debian BTS or salsa.debian.org URL
Author: Hideki Yamane <henrich@debian.org>
Forwarded: Yes
Bug-Debian: #927914

--- a/foobar.h
+++ b/foobar.h
@@ -20,6 +20,6 @@
 
 
 /**
- * You may know.
+ * You may know...
  */
 
